packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-phone/nodejs/integrationsteps.md

```diff
@@ -1 +1,74 @@
-Node.js sample code for this use case is coming soon. See [Okta Auth JS SDK](https://github.com/okta/okta-auth-js/blob/master/README.md) readme for further information.
\ No newline at end of file
+## Integration steps
+
+### Step 1: Build a sign-in page on the client and authenticate user credentials
+
+Build a sign-in page that captures the username and password, as shown in the following example.
+
+<div class="common-image-format">
+
+![Sign-in page](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-on-screenshot-sign-in.png
```

Comment:
```suggestion
![Sign-in page screenshot where user enters username and password for authentication.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-on-screenshot-sign-in.png)
```

Reply:
Chris pointed out that we got our alt-text in the wrong location. I've updated the style guide to reflect this.